core/controllers/base.py

```diff
@@ -425,6 +461,7 @@ def _render_exception(self, error_code, values):
                 self.render_template('pages/error/error.html')
 
     def handle_exception(self, exception, unused_debug_mode):
```

Comment:
`unused_debug_mode` this arg is not used in this function. Could you explain what it does? @seanlip